src/includes/performance/enable-tracing/java.spring-boot.mdx

```diff
@@ -28,6 +28,21 @@ import org.springframework.stereotype.Component;
 class CustomTracesSamplerCallback implements TracesSamplerCallback {
     @Override
     public Double sample(SamplingContext samplingContext) {
+        HttpServletRequest request = (HttpServletRequest) context.getCustomSamplingContext().get("request")
+        // return a number between 0 and 1
+    }
+}
+```
+
+```kotlin {tabTitle:Kotlin}
+import io.sentry.SentryOptions.TracesSamplerCallback
+import io.sentry.SamplingContext
+import org.springframework.stereotype.Component
+
+@Component
+class CustomTracesSamplerCallback : TracesSamplerCallback {
+    override fun sample(context: SamplingContext): Double {
+        val request = context.customSamplingContext!!["request"] as HttpServletRequest
```

Comment:
Is `!!` the way to go here?
Maybe we document with `?` instead to avoid having potential bugs on these snippets?

Stuff like this can get copy pasted in contexts where the request isn't available